docs/docs/explanations/advanced-custom-settings.md

```diff
@@ -288,7 +343,3 @@ conda_store:
       cpu: 1
       memory: 4Gi
 ```
```

Comment:
```suggestion
```

:::note Note
Current `conda_store.worker_resources` defaults are set at the minimum recommended resources for conda-store-workers - (conda-store [docs](https://conda.store/conda-store/references/faq#what-are-the-resource-requirements-for-conda-store-server))
:::
```